### DISCUSSION TITLE
Too many typos in written lesson

### DISCUSSION THREAD

**Discussion by vijethx** [Cyfrin/foundry-full-course-cu#1767] [General]
I was going through Blockchain Basics again to brush up before I started the next course, and that was when I noticed a lot of typos in the written lesson. 

How can I contribute by fixing these typos? I see the content displayed is not on the course repo

**Comment by EngrPips** [comment]
Hello @vijethx, You should use the `issues` tab next time for a discussion of this nature. You can re-write the written lesson and make a PR.

**Reply by vijethx** [reply]
I posted in discussions because that's where I was redirected from Issues tab

I didn't see the written content in the repo. Any idea where I can find it so that I can create a PR?

**Comment by EngrPips** [comment]
Oh, I see. I asked for the link, and I will get back to you as soon as I have it.

**Comment by EngrPips** [comment]
Here you go `https://github.com/Cyfrin/Updraft` sir. Also, check the pending PRs to see whether there are fixes already for what you want to make PR.
